function/Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM tensorflow/tensorflow:latest
+FROM tensorflow/tensorflow:latest-py3
 
 RUN apt-get update -qy \
  && apt-get install -qy \
-       python3-pip
+       python3-pip curl
+       
 RUN curl -sSL https://github.com/openfaas/faas/releases/download/0.7.0/fwatchdog > /usr/bin/fwatchdog \
```

Comment:
We can fetch this without `curl` now, please see openfaas/templates for how and send an update.